QUARKUS-280.md

```diff
@@ -0,0 +1,31 @@
+## QUARKUS-280 - Tech Preview support for Message tracing and Open Telemetry
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-280
+
+OpenTelemetry extension for Quarkus is a preferred future solution for the application's telemetry. The current OpenTracing library for telemetry from Smallrye is deprecated and will not be supported in the future.
+Quarkus brings three extenions related to OpenTelemetry:
+ - quarkus-opentelemetry
+ - quarkus-opentelemetry-exporter-jaeger
+ - quarkus-opentelemetry-exporter-otlp
+
+ OpenTelemetry is used to collect the application's telemetry data (metrics, logs, and traces) and to track how a particular user request is traversing through the different stages in a distributed system. Collected data are used to determine application performance and to investigate its behavior.
+
+ Quarkus Guide: https://quarkus.io/guides/opentelemetry  
+ Quarkus Guide upstream: https://github.com/quarkusio/quarkus/commits/main/docs/src/main/asciidoc/opentelemetry.adoc
+
+## Scope of the testing
```

Comment:
Our scope of work according the analysis in the rfe is 
AIs for RHBQ 2.2 / QE:
```
    New tests in QE TS to try OpenTelemetry on OpenShift
    Reach equivalent test coverage as for OpenTracing in QE TS
```
Please adjust the TP and link the analysis that I did.

Community side conclusions section says `Community test coverage needs to improve in both Quarkus integration tests and Quarkus Quickstarts (use Jaeger, cover Jaeger exporter, cover Reactive Messaging).`

This effort is for upstream developers, not us. I will create GH issues for these areas as requested by Thomas (https://issues.redhat.com/browse/QUARKUS-280?focusedCommentId=18936836&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-18936836  and bellow)